pkgs/development/libraries/LAStools/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, cmake, fetchurl}:
+stdenv.mkDerivation rec {
+
+  version = "180812";
+  name = "LAStools-${version}";
```

Comment:
* [Don’t use uppercase letters in the name attribute — e.g., "mplayer-1.0rc2" instead of "MPlayer-1.0rc2".](https://nixos.org/nixpkgs/manual/#sec-package-naming)
 * [If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".](https://nixos.org/nixpkgs/manual/#sec-package-naming)

Maybe their history was re-written, but I don't see any commit on 2018-08-12 (the commit is 2018-08-18). Unless I missed a releases page.

(And yes, partly nit picking here!)

Reply:
@samueldr ... I'd like to pick this up in #103253 because I agree with @mpickering